docs/tutorials/basic_networking.md

```diff
@@ -116,7 +107,7 @@ client can connect to the container.
 00f3440c7576aae2d5b193c40513c29c7964e96bf797cf0cc352c2b68ccbe66a
 ```
 
-As mentioned earlier, for rootless containers using CNI, a network must first be
+As mentioned earlier, for rootless containers using netavark, a network must first be
```

Comment:
No longer true, rootless also has a default bridge but you have to use `--network bridge` to use it because slirp4netns is default.